addons/common/functions/fnc_rscObjectHelper.sqf

```diff
@@ -0,0 +1,64 @@
+#include "script_component.hpp"
```

Comment:
I know we can't convert screen coords to 3d as `screenToWorld` does not account for objects, but can't 3d => 2d be done with `worldToScreen` command?

Reply:
this is for RscObjects
I don't think worldToScreen will work

Reply:
this is for the syntax 3 of https://community.bistudio.com/wiki/ctrlSetPosition which excepts a vec3